core/src/tezos/TezosLikeAddress.cpp

```diff
@@ -119,5 +119,43 @@ namespace ledger {
             }
             return std::make_shared<ledger::core::TezosLikeAddress>(currency, hash160, version, derivationPath);
         }
+
+        std::vector<uint8_t> TezosLikeAddress::getPublicKeyHash160(const std::vector<uint8_t> &pubKey,
+                                                                   api::TezosCurve curve) {
+            auto result = vector::concat(
+                    std::vector<uint8_t>{static_cast<uint8_t>(0x02 + (pubKey[64] & 0x01))},
+                    std::vector<uint8_t>{pubKey.begin() + 1, pubKey.begin() + 33}
+            );
+            auto sRes = hex::toString(result);
+            switch (curve) {
+                case api::TezosCurve::ED25519 :
+                    if (pubKey.size() != 33) {
+                        throw Exception(api::ErrorCode::INVALID_ARGUMENT, "Invalid ED25519 public key: should be 33 bytes.");
+                    }
+                    return BLAKE::blake2b(std::vector<uint8_t>{pubKey.begin() + 1, pubKey.end()}, 20);
```

Comment:
Just to be sure, why are you skipping the first byte ? Can you add a comment to explain this technical choice ?